index.rst

```diff
@@ -471,6 +494,42 @@ retrieved by adding ``_`` plus the extension to the dataset type when calling
 * ``<datasetType>_visitInfo`` for example
   ``wcs = butler.get(calexp_visitInfo, ...)``
 
+Bypass Functions
+^^^^^^^^^^^^^^^^
+
+If a ``CameraMapper`` subclass wants to use a different deserializer than the
+standard Butler deserialization schemes, it can implement a function that starts
+with ``bypass`` and ends with the datasetType name. The function signature must
```

Comment:
Maybe "`bypass_` followed by datasetType name"? Or is it literally "bypass" and datasetType with anything in between?